client/src/pages/SubmitDesign.js

```diff
@@ -115,6 +117,13 @@ export default function SubmitDesign(props) {
         })
     }
 
+    if (user.username === "no-user") {
+        return (
+          <Redirect to='/login'/>
```

Comment:
hm with this, if user doesn't load just yet, will this redirect to login? for example if you refresh this submit design page, the user i'm guessing won't load as fast as this page gets rendered